db/migrate/20220428214622_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [x] As per the given ERD, the _Author_ model is acting as a through model association to _User_ model, hence implementing a foreign key by creating _author_id_ and a corresponding _index_ should be similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Kindly ensure to improve upon this aspect.